uvloop/loop.pyx

```diff
@@ -821,13 +824,17 @@ cdef class Loop:
                 except Exception as ex:
                     if not fut.cancelled():
                         fut.set_exception(ex)
+                    __send_trace(TRACE_DNS_REQUEST_END, None)
                 else:
                     if not fut.cancelled():
                         fut.set_result(data)
+                    __send_trace(TRACE_DNS_REQUEST_END, data)
```

Comment:
unfortunately, the `_getaddrinfo` can be called with the `unpack` enabled or disabled. with the current PR, the user won't be able to know explicitly wich format meets the argument.

I more inclined to think that we must give always the unpack version of the result that is the format returned by the public method `getaddrinfo`.